metadata_layer.py

```diff
@@ -1047,15 +1064,15 @@ def __init__(
         self.links = links
 
         # Information about individual terms for each license in a pool.
-        self.licenses = licenses or []
+        self.licenses = licenses
```

Comment:
This is one of the major changes in this PR. If `CirculationData` gets licenses, it will use those licenses to calculate availability, rather then using `licenses_owned`, `licenses_available`, `licenses_reserved` and `patrons_in_hold_queue`. If it does not, it defaults to the previous behaviour.